src/dbg/debugger_commands.cpp

```diff
@@ -1997,6 +2002,41 @@ CMDRESULT cbDebugKillthread(int argc, char* argv[])
     return STATUS_ERROR;
 }
 
+CMDRESULT cbDebugCreatethread(int argc, char* argv[])
+{
+    if(argc < 2)
+        return STATUS_ERROR;
+    duint Entry = 0;
+    duint Argument = 0;
+    if(!valfromstring(argv[1], &Entry))
+        return STATUS_ERROR;
+    if(!MemIsCodePage(Entry, false))
+        return STATUS_ERROR;
+    if(argc > 2)
+    {
+        if(!valfromstring(argv[2], &Argument))
+            return STATUS_ERROR;
+    }
+    DWORD ThreadId = 0;
+    if(ThreaderCreateRemoteThread(Entry, true, reinterpret_cast<LPVOID>(Argument), &ThreadId) != 0)

```

Comment:
It doesn't have to be because it will be informed with the `CREATE_THREAD_DEBUG_EVENT`, the code for `ThreaderCreateRemoteThread` is:

```
__declspec(dllexport) ULONG_PTR TITCALL ThreaderCreateRemoteThreadEx(HANDLE hProcess, ULONG_PTR ThreadStartAddress, bool AutoCloseTheHandle, LPVOID ThreadPassParameter, LPDWORD ThreadId)
{
    if(hProcess != NULL)
    {
        if(!AutoCloseTheHandle)
        {
            return (ULONG_PTR)CreateRemoteThread(hProcess, NULL, NULL, (LPTHREAD_START_ROUTINE)ThreadStartAddress, ThreadPassParameter, NULL, ThreadId);
        }
        else
        {
            HANDLE myThread = CreateRemoteThread(hProcess, NULL, NULL, (LPTHREAD_START_ROUTINE)ThreadStartAddress, ThreadPassParameter, NULL, ThreadId);
            EngineCloseHandle(myThread);
            return NULL;
        }
    }
    return NULL;
}
```